terraform/environments/dev/main.tf

```diff
@@ -0,0 +1,84 @@
+locals {
+  project_owners = [
+    "a@firezone.dev",
+    "bmanifold@firezone.dev",
+    "gabriel@firezone.dev",
+    "jamil@firezone.dev",
+    "thomas@firezone.dev"
+  ]
+
+  region            = "us-east1"
+  availability_zone = "us-east1-d"
+
+  tld = "firez.one"
+}
+
+terraform {
+  cloud {
+    organization = "firezone"
+    hostname     = "app.terraform.io"
+
+    workspaces {
+      name = "dev"
+    }
+  }
+}
+
+provider "random" {}
+provider "null" {}
+provider "google" {}
+provider "google-beta" {}
+
+# Create the project
+module "google-cloud-project" {
+  source = "../../modules/google-cloud-project"
```

Comment:
@AndrewDryga Anything else we should add in here? I didn't spend a ton of time looking through our staging env, so I could be missing something.

Reply:
I'll add this to my PR with prod env.

Reply:
Yeah, I'm not opposed to that, but can we override the project's public access policy if needed?

https://cloud.google.com/storage/docs/public-access-prevention

Reply:
@jamilbk I'll just create a proper ACL policy for that bucket